创建本地文件存储数据库

0xBB2B opened the pull request.

Commit:
add File-based store init

Commit:
add File-based store init

Commit:
add chat model

Commit:
add chat model

Commit:
add SubscribedChats

Commit:
add SubscribedChats

Commit:
add bolt with file store

Commit:
add bolt with file store

need-being commented on pkg/storage/chat.go:
```diff
@@ -0,0 +1,23 @@
+
+// FromJSON is
+func (c *Chat) FromJSON(jsonBytes []byte) {
+	json.Unmarshal(jsonBytes, c)
```
如果解码出错怎么办？

need-being commented on pkg/storage/chat.go:
```diff
@@ -0,0 +1,23 @@
+// Chat is
+type Chat struct {
+	ID             string
+	SubscribedApps []App
```
Chat套App，App套Chat。循环了

need-being commented on pkg/storage/app.go:
```diff
@@ -3,15 +3,31 @@ package storage
+
+// FromJSON is
+func (a *App) FromJSON(jsonBytes []byte) {
+	json.Unmarshal(jsonBytes, a)
```
如果解码出错怎么办？

Commit:
changed FromJSON() return type
changed App SubscribedChatIDs to string
changed Chat SubscribedAppIDs to string

Commit:
changed FromJSON() return type
changed App SubscribedChatIDs to string
changed Chat SubscribedAppIDs to string

Commit:
Add comment

Commit:
Add comment

Commit:
Completion CreateApp func

Commit:
Completion CreateApp func

Commit:
Correct AppStore.CreateApp Interface (#13)

Fixes #12

Commit:
Merge branch 'bb/store' of github.com:ne2blink/antenna into bb/store
